pkgs/development/python-modules/pythonocc-core/default.nix

```diff
@@ -12,11 +12,14 @@
 , libXext
 , libXi
 , libXmu
-, opencascade-occt
+, opencascade-occt_7_6
```

Comment:
should such overrides be done in python-packages.nix ?

https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#changing-implicit-attribute-defaults

> would change the .override interface

Reply:
Note the documentation you link to is for 'by-name'.

Requesting the specific version is consistent with `ab2d6f785322ca20cf62779ec206539721d3ba4a`.

There's something to be said for either approach: indeed putting it in `default.nix` changes the override interface, on the other hand, the requirement to have a 7.6 occt is arguably 'local to pythonocc-core', which makes putting it in `python-packages.nix` instead awkward as well.

Let's try converging on the same convention as in by-name though - updated.